examples/cisco/8000e/8000e-ixia.pb.txt

```diff
@@ -0,0 +1,130 @@
+name: "cisco-ixia"
+nodes: {
+    name: "vxr"
+    vendor: CISCO
+    model: "8201"
+    os: "ios-xr"
+    config: {
+        image: "ios-xr:latest" # please update the image
```

Comment:
this can be omitted, we have a default value for the image defined in the node implementation which is good enough for an open source example

if a user has access to an image they can add the image field to point to wherever it is located

Reply:
done